src/stories/Button/Button.tsx

```diff
@@ -0,0 +1 @@
+export * from '../../components/Button';
```

Comment:
You can import `Button` directly w/o re-export

Reply:
Fix